cogs/Tasks.py

```diff
@@ -0,0 +1,97 @@
+import discord, json, datetime
+from discord.ext import commands
+from discord.ext import tasks
+
+from cogs.Moderation import Moderation
+
+class Tasks(commands.Cog):
+
+    def __init__(self, client, config):
+        self.client = client
+        self.config = config
+        self.scheduler.start()
+
+    @staticmethod
+    def isNow(cronStamp):
+        now = datetime.datetime.now()
+        cronStamp = cronStamp.split()
+        
+        # Minutes
+        if cronStamp[0] != "*":
+            nowMins = int(now.strftime("%M"))
+            if int(cronStamp[0]) != nowMins:
+                return False
+        # Hour
+        if cronStamp[1] != "*":
+            nowHours = int(now.strftime("%H"))
+            if int(cronStamp[1]) != nowHours:
+                return False
+
+        # Date
+        if cronStamp[2] != "*":
+            nowDate = int(now.strftime("%d"))
+            if int(cronStamp[2]) != nowDate:
+                return False
+
+        # Month
+        if cronStamp[3] != "*":
+            nowMonth = int(now.strftime("%m"))
+            if int(cronStamp[3]) != nowMonth:
+                return False
+
+        # Day
+        if cronStamp[4] != "*":
+            nowDay = int(now.strftime("%w"))
+            if int(cronStamp[4]) != nowDay:
+                return False
+
+        return True
+
+##    @commands.command("test")
+##    async def test(self, msg):
+##        await Moderation.lock(self, msg, "comp1000")
+
+    
+    @tasks.loop(minutes=1.0)
+    async def scheduler(self):
+        # Reads file in the same format as crontab
+        # Minute Hour Date Month Day
+        f = open("tasks.json")
```

Comment:
Should check if file exists. Also would ideally "register" a filename so that multiple different files can be run simultaneously without having to append everything to a single one